fuel/downloaders/base.py

```diff
@@ -55,6 +69,9 @@ def download(url, file_handle, chunk_size=1024):
         URL to download.
     file_handle : file
         Where to save the downloaded URL.
+    fd : open file handle, optional
```

Comment:
After some thought, the only reason I have this kwarg is so that I don't litter the test logs with progress bar outputs (nose buffers stdout and nose2 can do the same with the -B flag). I just changed the `fd` argument to `ProgressBar` to be `sys.stdout` and we'll use that as default.